examples/servers/reading-list/.kube.yml

```diff
@@ -0,0 +1,97 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: reading-list-server
+spec:
+  minReadySeconds: 30
+  replicas: 5
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 2
+  template:
+    metadata:
+      labels:
+        app: reading-list-server
+    spec:
+      terminationGracePeriodSeconds: 60
+      volumes:
+      - name: nyt-reading-list-tls 
+        secret:
+          secretName: nyt-reading-list-tls
+      containers:
+        - name: esp
+          image: "gcr.io/endpoints-release/endpoints-runtime:1.3.0"
+          args: [
+            "-S", "443",
```

Comment:
Without HTTPS, gRPC is pointless tho!

Folks can still run this locally if the want to test it out and if they want to put it on GKE for realz, they can use `lego` to put the certs into the `nyt-reading-list-tls` secret: https://github.com/jetstack/kube-lego

This setup is also a mirror of the recommended kube config for CE: https://cloud.google.com/endpoints/docs/enabling-ssl#enabling_ssl_for_service_proxy_on_kubernetes